extensions/jms/src/main/java/org/mule/extensions/jms/JmsExtension.java

```diff
@@ -47,6 +49,7 @@
 @Xml(prefix = "jmsn")
```

Comment:
old transport is already out of the distribution. Confirm with @elrodro83 if it's ok to change this to simply `jms`